### PR TITLE
Reorganize Contributors Guide

### PR DESCRIPTION
**Pull request by bmmpxf**
This PR cleans up the Contributing section, deletes unneeded files, and moves most of the content from the README.md to this section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,200 +1,35 @@
...
-name: Avalanche C-Chain
-spec_url: https://path-to-avax/spec.json
-portal_prefix: avax-mainnet
-ignore_api_path: true # Warning: explicitly setting to false still ignores path
-api_slug: /ext/bc/C/
----
-```
-
-## Customizing Content
+4. View the built docs in your browser at http://localhost:1131.
```
Comment (blockjoe) [inline README.md:28]: Should this be localhost:1313? At least that's what mine has always defaulted to.

**Review by blockjoe — CHANGES_REQUESTED** [review]
This all looks so much cleaner!

Only thing that jumped at me was the localhost:1131 on the README

File: README.md (hunk 1 above)
Comment (bmmpxf) [reply README.md:28]: Great catch, yes, that's incorrect.

Comment (bmmpxf) [reply README.md]: Fixed now.

**Review by blockjoe — APPROVED** [review]
🎆

**Pull request merged**